.readthedocs.yml

```diff
@@ -4,4 +4,6 @@ mkdocs:
   configuration: mkdocs.yml
 
 python:
-  version: 3.6
+  version: 3.8
```

Comment:
Currently this project has the following in the settings part at RTD admin web:

![imagen](https://user-images.githubusercontent.com/1534240/165514712-c23cf1a8-0a0c-4e8d-9105-6546d90cf348.png)

I understand that the settings in .readthedocs.yml override the ones in the RTD admin web. Is this correct? Or it is a not related setting?

Reply:
Yes, the project file will override the defaults.  

Is anyone still using Python 2?

Reply:
> Is anyone still using Python 2?

Nobody *should* use Python 2, but for sure a lot of places (legacy, mostly) are still using it.

Not sure about reliable usage statistics on Python versions, but if we look to:

* https://w3techs.com/technologies/details/pl-python/2 (0.5%)
* https://w3techs.com/technologies/details/pl-python/3 (0.8%)

Based on that, Python 3 would be 1.6 times more used that Python 2... which would mean that a lot of legacy is still there today (a good ratio would be 10 or even 100 times ;)

Reply:
NTC, anycase